### PR TITLE
refactor/510

### LINKED ISSUE #510: Remove mapping of -/+/= to zoom in/out/reset
Use of `-`, `+` and `=` are used both for speed control as well as zooming the canvas. The latter is not needed, since we can scroll to achieve the desired zoom level.

Remove logic that uses these key events to trigger zooming.

### PR DESCRIPTION
**Pull request by sam-irl**
Remove +/-/= as hotkeys for zoom as it conflicts with speed control, fixes #510

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,9 +12,10 @@
 
 
 ### Bugfixes
--Removes inactive danger areas at EIDW [#562](https://github.com/openscope/openscope/issues/562)
+- Removes inactive danger areas at EIDW [#562](https://github.com/openscope/openscope/issues/562)
```
Comment (n8rzz) [inline CHANGELOG.md]: Be sure this branch includes only the changes for this issue.  This Changelog entry looks like it's from your other branch, please remove it.

Comment (sam-irl) [inline CHANGELOG.md]: sorry, will remove. i just noticed the improper formatting.

Comment (n8rzz) [inline CHANGELOG.md]: no worries.

It looks like there is also an open invite out there for you to join the organization.  Could you accept that?  Then all future changes can be made in this repo which will avoid the need for a fork and allow our tests to run hen you open a PR.

You can also join us on slack for faster communication http://slack.openscope.co

**Comment by indianbhaji** [comment]
@sam-irl You don't approve your own changes though, there's no point in approving your own changes, not everything is perfect every time

**Comment by sam-irl** [comment]
@indianbhaji Yeah, sorry, that was phrased poorly. I meant that I've made the changes and in future I won't need to open the PR.

**Comment by n8rzz** [comment]
typically, we just re-request a review.  You can do that by un-checking and re-checking a reviewer in the top right of the PR page.

**Pull request merged**